packages/experimental/Badge/src/Badge.tsx

```diff
@@ -13,13 +13,11 @@ export const badgeLookup = (layer: string, userProps: BadgeProps): boolean => {
   return (
     userProps[layer] ||
     layer === userProps['appearance'] ||
-    (!userProps['appearance'] && layer === 'filled') ||
```

Comment:
I had to remove these values because in this case customization was only possible by overriding `brand: { filled: { ...styles } }`